### PR TITLE
Review pre-release

### PR DESCRIPTION
**Pull request by vhf**
~~I'll rebase on top of #1 after it gets merged. :)~~

done

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,20 +7,20 @@
...
-The function accepts the same parameters like [fetch](https://fetch.spec.whatwg.org/), but also accepts some addition options and provides additional methods.
+The package exports a fetch function which wraps the request and response object to process RDF quads on the fly.
+The function accepts the same parameters as [fetch](https://fetch.spec.whatwg.org/) and some additional options. It also provides additional methods.
 
 ### Options
 
 The `options` object accepts the following additional parameters:
 
-- `formats`: An [formats-common](https://github.com/rdfjs/formats-common) compatible object which contains a set of parsers and serializers.
+- `formats`: A [formats-common](https://github.com/rdfjs/formats-common) compatible object which contains a set of parsers and serializers.
```
Comment (vhf) [inline README.md]: An example would be good here, it's not clear what a formats-common compatible object is.

File: .travis.yml [hunk 2]
```diff
@@ -1,13 +1,4 @@
...
   - "8"
   - "10"
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - g++-4.9
-before_install:
-  - export CXX="g++-4.9" CC="gcc-4.9"
```
Comment (vhf) [inline .travis.yml]: Not sure if all of this is still needed, looking at the deps/devDeps I guess not.

Comment (bergos) [reply .travis.yml]: That's for JSON-LD, which is part of the formats-common package. I will have a look.

Comment (vhf) [reply .travis.yml]: Travis is happy, no need to spend more time on this :)

File: README.md [hunk 3]
```diff
@@ -29,22 +30,21 @@ The `options` object accepts the following additional parameters:
...
+  Iterables will be converted to streams before they are handed over to the serializer.
 
 ### Response
 
 The following methods are attached to the standard fetch response object:
 
 - `quadStream()`: This method returns the quads of the response as stream.
   The parser is selected based on the content type header field.
-- `dataset()`: This method uses the `quadStream()` method to parse the content and will pipe it into a dataset, which is also the return value.
-  The method is async, so `await` is required to get the actual result.
```
Comment (vhf) [inline README.md]: I indicated `async` on the previous line to avoid having to mention `.then` etc

**Comment by vhf** [comment]
@bergos Feel free to merge :)

**Pull request merged**